app/components/offering-editor.js

```diff
@@ -0,0 +1,190 @@
+/* global moment */
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const { Component, computed, isEmpty, ObjectProxy, RSVP } = Ember;
+const { notEmpty } = computed;
+const { all, Promise } = RSVP;
+const { PromiseArray } = DS;
+
+export default Component.extend({
+  init() {
+    this._super(...arguments);
+    this.setProperties({ instructors: [], instructorGroups: [], learnerGroups: [] });
+  },
+
+  classNames: ['offering-editor'],
+
+  singleOffering: true,
+  isMultiDay: false,
+  filter: '',
+
+  startDate: new Date().setHours(8, 0, 0),
```

Comment:
These properties should probably be called defaultStartDate / defaultEndDate unless they are getting set somewhere?